docs/getting_started.rst

```diff
@@ -0,0 +1,18 @@
+Getting Started
+===============
+
+If you have not already done so, create/activate a `virtualenv`_. Unless otherwise stated, assume all terminal code
+below is executed within the virtualenv.
+
+.. _virtualenv: https://virtualenvwrapper.readthedocs.org/en/latest/
```

Comment:
i suppose there is _this_ reference to virtualenv wrapper 🤔 -- i dunno, i bet the average person (like me!) is going to look at the readme and not docs/getting_started.rst so I still keep my ask to add that alternatve VE command =P